lib/CustomFields/pages/ViewCustomFieldsSettings/ViewCustomFieldsSettings.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import {
+  Pane,
+  Button,
+  Icon,
+  AccordionSet,
+  KeyValue,
+} from '@folio/stripes-components';
+
+import {
+  FieldAccordion,
+  NoCustomFieldsMessage,
+} from '../../components';
+
+import { useCustomFieldsFetch } from '../../utils';
+
+import {
+  selectModuleId,
+  selectOkapiData,
+} from '../../selectors';
+
+import styles from './ViewCustomFieldsSettings.css';
+
+const propTypes = {
+  backendModuleId: PropTypes.string,
+  backendModuleName: PropTypes.string.isRequired, // eslint-disable-line
+  okapi: PropTypes.object.isRequired,
+  redirectToEdit: PropTypes.func.isRequired,
+};
+
+const ViewCustomFieldsSettings = ({ redirectToEdit, okapi, backendModuleId }) => {
+  const { customFields, customFieldsLoaded } = useCustomFieldsFetch(okapi, backendModuleId);
+
+  const renderLastMenu = () => (
+    <Button
+      buttonStyle="primary"
+      marginBottom0
+      onClick={redirectToEdit}
+    >
+      {customFields.length
```

Comment:
Please use lodash' `isEmpty` for this. Same [here](https://github.com/folio-org/stripes-smart-components/pull/703/files#diff-fbb0b871db96bba556083674d43d8318R85), [here](https://github.com/folio-org/stripes-smart-components/pull/703/files#diff-7662134690a4a418782d3d9eb9a70961R208), [here](https://github.com/folio-org/stripes-smart-components/pull/703/files#diff-7662134690a4a418782d3d9eb9a70961R227)
```suggestion
      {isEmpty(customFields)
```